launchable/commands/record/commit.py

```diff
@@ -54,14 +58,14 @@ def exec_jar(source, max_days, dry_run):
     https_proxy = os.getenv("HTTPS_PROXY")
     proxy_option = _build_proxy_option(https_proxy) if https_proxy else ""
 
-    os.system(
-        "{} {} -jar \"{}\" ingest:commit -endpoint {} -max-days {} {} {} {} {}"
-        .format(java, proxy_option, jar_file_path,
-                "{}/intake/".format(base_url), max_days,
-                "-audit" if Logger().logger.isEnabledFor(LOG_LEVEL_AUDIT) else "",
-                "-scrub-pii",
-                "-dry-run" if dry_run else "",
-                source))
+    subprocess.check_output("{} {} -jar \"{}\" ingest:commit -endpoint {} -max-days {} {} {} {} {}"
```

Comment:
[nits]
knowing this is not the aim of this PR, you can write python string formatter to have placement with variable within strings to specify which placement corresponds to which value. This is useful for better readability in case there is a lot of placement than positional placement.

```python
subprocess.check_output("{java} {proxy_option} -jar \"{jar_file_path}\" ingest:commit -endpoint {base_url}/{intake}/ -max-days {max_days} {audit} {scrub_pii} {dry_run} {source}"
    .format(
        java=java, 
        proxy_option=proxy_option, 
        jar_file_path=jar_file_path,
        base_url=base_url,
        intake="intake", 
        max_days=max_days,
        audit="-audit" if Logger().logger.isEnabledFor(LOG_LEVEL_AUDIT) else "",
        scrub_pii="-scrub-pii",
        dry_run="-dry-run" if dry_run else "",
        source=source,
    ), 
    shell=True)
```

Reply:
TIL, I'll fix it